src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -179,8 +179,9 @@ function MyServices(): React.JSX.Element {
             {
                 key: 'details',
                 label:
-                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DESTROY_SUCCESSFUL ||
-                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DEPLOYMENT_FAILED ? (
+                    record.serviceHostingType === DeployedService.serviceHostingType.SERVICE_VENDOR &&
```

Comment:
@WangLiNaruto as discussed, there should be no logic based on service state. It must be based on `details` coming from backend. 

Also, if the fix will take a bit more time, we can consider doing a quick PR to revert the first fix and then work on the actual final fix.